protobuf/flow/entities/block_header.proto

```diff
@@ -11,11 +11,12 @@ message BlockHeader {
   bytes id = 1;
   bytes parent_id = 2;
   uint64 height = 3;
-  bytes payload_hash = 4;
-  google.protobuf.Timestamp timestamp = 5;
+  google.protobuf.Timestamp timestamp = 4;
+  bytes payload_hash = 5;
```

Comment:
With protobuf, one shouldn't change the field numbers for any existing fields as it would break old clients.

See the [Updating a Message Type](https://developers.google.com/protocol-buffers/docs/proto#updating) documentation for more info.

Reply:
@tav Thanks, as I mentioned in my comment above: https://github.com/onflow/flow/pull/843#pullrequestreview-891943697

This is actually a change that was mistakenly made a previous PR, and this PR is reverting the order change.